include/seqan3/utility/views/enforce_random_access.hpp

```diff
@@ -73,7 +73,7 @@ class view_enforce_random_access : public std::ranges::view_interface<view_enfor
               std::ranges::viewable_range<viewable_rng_t> &&
               std::constructible_from<urng_t, std::ranges::ref_view<std::remove_reference_t<viewable_rng_t>>>
     //!\endcond
-    explicit constexpr view_enforce_random_access(viewable_rng_t range) :
+    explicit constexpr view_enforce_random_access(viewable_rng_t && range) :
         view_enforce_random_access{std::views::all(range)}
```

Comment:
könnte hier auch ein std::move hin?
```suggestion
        view_enforce_random_access{std::views::all(range)}
```

Reply:
Could, but apparently doesn't have to, I added a `std::forward`